frame/system/src/lib.rs

```diff
@@ -1039,6 +1055,15 @@ impl<T: Trait> Module<T> {
 	}
 }
 
+impl<T: Trait> sp_runtime::traits::Dispatcher<T::Call> for Module<T> {
```

Comment:
maybe you can move the `Call: Dispatchable` here (or implement `Dispatchable` for `()` that panics). 

Also, don't quote me on this, but I think there is a reason that we don't bound call to dispatchable so far. I have to think about it more but I am not sure if it complies with our abstractions.

Reply:
Yeah even if we keep the trait as-is, an impl of dispatchable for `()` would be nice. 

or some other way to not force all tests to define a Call.

Reply:
I think that is a huge pain point of this PR. I try to work around it starting with your suggestion.